components/service-binding-usage-controller/docs/proposals/usage-kind.md

```diff
@@ -37,15 +37,15 @@ UI needs two new endpoints which return:
  * all resources with a given kind. The result must be filtered out by the **metadata.ownerReference** field. If the resource contains **metadata.ownerReference**, the user should not see such a resource in the UI.
 ## Binding Usage Controller
 
-The Binding Usage Controller takes the value of the **spec.usedBy.kind** field of the ServiceBindingUsage resource, looks for the corresponding UsageKind which contains information about the kind of resource to be bound. The controller must handle resources even if the specified **labelsPath** field does not exist.
+The Binding Usage Controller takes the value of the **spec.usedBy.kind** field of the Service Binding Usage resource, looks for the corresponding Usage Kind which contains information about the kind of resource to be bound. The controller must handle resources even if the specified **labelsPath** field does not exist.
 
 ## Security
 
-The administrator who adds UsageKind must take care of RBAC settings. BUC must be allowed to perform needed operations on resources with the kind defined in the UsageKind resource.
+The administrator who adds Usage Kind must take care of RBAC settings. BUC must be allowed to perform needed operations on resources with the kind defined in the Usage Kind resource.
 
 ## Example
 
-There is a defined UsageKind:
+There is a defined Usage Kind:
 ```yaml
 apiVersion: servicecatalog.kyma-project.io/v1alpha1
 kind: UsageKind
```

Comment:
I just noticed line 68 talks about `Console Backend Service`, but AFAIK, it's been removed by the Busola team. 
Can you please check with the tea, see if the documentation must be updated, and if so, create an issue for it?

Reply:
Will do :)